### ISSUE TITLE
Can you post the training script?

### ISSUE THREAD

**Issue by ts19970816** [herbwood/pytorch-mask-rcnn#1]

**Comment by aggelos-michael-papadopoulos** [comment]
yeah that would be awesome if you did
